spec/acceptance/suites/rpm_docker/nodesets/default.yml

```diff
@@ -0,0 +1,129 @@
+HOSTS:
+  el7-build-server:
+    roles:
+      - primary
+    platform: el-7-x86_64
+    hypervisor: docker
+    # This needs to be the *oldest* image of EL7 to preserve SELinux build compatibility
+    image: centos:7.0.1406
+    docker_cmd: '/usr/sbin/sshd -D'
+    docker_preserve_image: true
+    docker_image_commands:
+      # For SELinux Builds
+      - 'yum install -y yum-utils'
+
+      # We only want to deal with the original distro packages
+      - 'yum-config-manager --disable \*'
+      - 'echo -e "[legacy]\nname=Legacy\nbaseurl=http://vault.centos.org/7.0.1406/os/x86_64\ngpgkey=https://www.centos.org/keys/RPM-GPG-KEY-CentOS-7\ngpgcheck=1" > /etc/yum.repos.d/legacy.repo'
+
+      # Getting rid of package conflicts
+      - 'yum remove -y fakesystemd'
+      - '\cp -a /etc/ssh /root'
+      - 'yum downgrade -y openssh openssh-clients systemd-libs'
+      - 'yum install -y openssh-server'
+      - '\cp -a /root/ssh /etc'
+
+      # Add the SELinux Build dependencies
+      - 'yum install -y selinux-policy-targeted selinux-policy-devel policycoreutils policycoreutils-python'
+
+      # Allow the build user to perform privileged operations
+      - "echo 'Defaults:build_user !requiretty' >> /etc/sudoers"
+      - "echo 'build_user ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers"
+
+      # Standard mock group for compatibility with legacy builds
```

Comment:
Why do we still care about legacy builds? Users should be able to checkout previous versions of simp-core if they want old builds. Otherwise, this is cruft.

Reply:
Good call